packages/manager/modules/exchange/src/domain/spf-autoconfig/domain-spf-autoconfig.controller.js

```diff
@@ -0,0 +1,111 @@
+export default class ExchangeDomainSpfAutoconfigCtrl {
+  /* @ngInject */
+  constructor(
+    $scope,
+    wucExchange,
+    ExchangeDomains,
+    navigation,
+    messaging,
+    $translate,
+    exchangeStates,
+  ) {
+    this.services = {
+      $scope,
+      wucExchange,
+      ExchangeDomains,
+      navigation,
+      messaging,
+      $translate,
+      exchangeStates,
+    };
+
+    this.$routerParams = wucExchange.getParams();
+    this.domain = navigation.currentActionData;
+    this.domainDiag = {};
```

Comment:
```suggestion
    this.domainDialog = {};
```

Reply:
domainDiag was not my choice, was already defined like that before.
For info, it's not mean Dialog but Diagnose ;)